# HG changeset patch
# User Jeroen Demeyer <jdemeyer@cage.ugent.be>
# Date 1306231886 -10800
# Node ID 611e83c50dc9bb56a04a52ae752b118a36fd0213
# Parent  5655998908b8797a2108b8e8cac38d705faa3804
Allow global extra_link_args just like extra_compile_args

diff -r 5655998908b8 -r 611e83c50dc9 setup.py
--- a/setup.py	Sat May 21 08:20:37 2011 +0200
+++ b/setup.py	Tue May 24 13:11:26 2011 +0300
@@ -114,6 +114,7 @@
     include_dirs = include_dirs + debian_include_dirs
 
 extra_compile_args = [ ]
+extra_link_args = [ ]
 
 # comment these four lines out to turn on warnings from gcc
 import distutils.sysconfig
@@ -203,7 +204,8 @@
 
 for m in ext_modules:
     m.libraries = ['csage'] + m.libraries + ['stdc++', 'ntl']
-    m.extra_compile_args += extra_compile_args # + ["-DCYTHON_REFNANNY"]
+    m.extra_compile_args += extra_compile_args
+    m.extra_link_args += extra_link_args
     if os.environ.has_key('SAGE_DEBIAN'):
         m.library_dirs += ['/usr/lib','/usr/lib/eclib','/usr/lib/singular','/usr/lib/R/lib','%s/lib' % SAGE_LOCAL]
     else:
